code/game/machinery/recharger.dm

```diff
@@ -25,9 +25,6 @@ obj/machinery/recharger
 	return
 
 /obj/machinery/recharger/attackby(obj/item/weapon/G as obj, mob/user as mob)
-	if(istype(user,/mob/living/silicon))
```

Comment:
I think this might screw with things like the secborg's baton, did/could you check that?

Reply:
You are correct. I'll have to fix that.

Reply:
Fixed with latest pull